torch/_inductor/fx_passes/quantization.py

```diff
@@ -1711,7 +1715,11 @@ def _generate_dequant_bmm_node_pattern(
         aten.bmm.default,
         CallFunction(
             aten.expand.default,
-            dequantize_per_tensor_activation_pattern,
+            _may_generate_pattern_with_dtype_convert(
+                dequantize_per_tensor_activation_pattern,
+                KeywordArg("autocast_act_dtype"),
+                dtype != torch.float32,
```

Comment:
do we support both bf16 and fp16 or only bf16?

Reply:
I think we only support int8_bf16 now. For int8_fp16, it should be a new path and we need further enabling and verification.

Reply:
Then shall we explicitly check `dtype == torch.bfloat16` here instead?

Reply:
Thanks for the comment.

- Add the check of `assert dtype in [torch.float32, torch.bfloat16]`
- Change `dtype != torch.float32` to `dtype == torch.bfloat16`

Please help to take a look again.